cla_public/apps/checker/tests/test_sessions.py

```diff
@@ -0,0 +1,130 @@
+        expectedJSON = json.dumps(expectedDict)
+        self.assert_json(jsonOutput)
+        self.assertEqual(jsonOutput, expectedDict)
+
+    def test_serializer_with_bytes(self):
+        value = b'test1'
+        jsonOutput = self.serializer.dumps(value)
+        expectedDict = {" b": b64encode(value).decode("ascii")}
+        expectedJSON = self.format_json(expectedDict)
+        self.assert_json(jsonOutput)
+        self.assertEqual(jsonOutput, expectedJSON)
+
+    def test_serializer_with_markup(self):
+        class TestMarkup:
+            def __html__(self):
+                return '<h1>Test</h1>'
+
+        value = TestMarkup()
+        jsonOutput = self.serializer.dumps(value)
+        expectedDict = {" m": text_type('<h1>Test</h1>')}
+        expectedJSON = self.format_json(expectedDict)
+        self.assert_json(jsonOutput)
+        self.assertEqual(jsonOutput, expectedJSON)
+
+    def test_serializer_with_list(self):
+        value = ["test", {"key1": "value2"}]
+        jsonOutput = self.serializer.dumps(value)
+        expectedDict = [{" b": b64encode("test").decode("ascii")}, {"key1": {" b": b64encode("value2").decode("ascii")}}]
+        expectedJSON = self.format_json(expectedDict)
+        self.assert_json(jsonOutput)
+        self.assertEqual(jsonOutput, expectedJSON)
+
+    def test_serializer_with_datetime(self):
+        value = datetime(1990, 1, 2)
+        jsonOutput = self.serializer.dumps(value)
+        expectedDict = {" d": http_date(value)}
+        expectedJSON = self.format_json(expectedDict)
+        self.assert_json(jsonOutput)
+        self.assertEqual(jsonOutput, expectedJSON)
+
+    def test_serializer_with_dict(self):
+        value = {"test": "string"}
+        jsonOutput = self.serializer.dumps(value)
+        expectedDict = {"test": {" b": b64encode("string").decode("ascii")}}
+        expectedJSON = self.format_json(expectedDict)
+        self.assert_json(jsonOutput)
+        self.assertEqual(jsonOutput, expectedJSON)
+
+    def test_serializer_with_str(self):
+        value = "hello"
+        jsonOutput = self.serializer.dumps(value)
+        expectedDict = {" b": b64encode(value).decode("ascii")}
+        expectedJSON = self.format_json(expectedDict)
+        self.assert_json(jsonOutput)
+        self.assertEqual(jsonOutput, expectedJSON)
+
+    def test_serializer_with_int(self):
+        value = 5
+        jsonOutput = self.serializer.dumps(value)
+        expectedValue = 5
+        expectedJSON = self.format_json(expectedValue)
+        self.assert_json(jsonOutput)
+        self.assertEqual(jsonOutput, expectedJSON)
+
+    def test_serializer_with_unicode_str(self):
+        value = u"hello"
+        jsonOutput = self.serializer.dumps(value)
+        expectedValue = u"hello"
+        expectedJSON = self.format_json(expectedValue)
+        self.assert_json(jsonOutput)
+        self.assertEqual(jsonOutput, expectedJSON)
```

Comment:
Wasn't sure if I needed to write tests for this, but I decided to anyway for the sake of making sure my refactoring works as expected..